### PR TITLE
Added 'ahoy debug' command.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
Resolves hectic workflow of modifying `docker-compose.yml` file any time Xdebug session is required.

- Run `ahoy debug` to enable debug session (this will restart containers).
- Run `ahoy up` to disable debug session (this will restart containers).

### CODE REVIEW

File: .ahoy.yml [hunk 1]
```diff
@@ -128,6 +128,12 @@ commands:
     usage: Run site audit
     cmd: docker-compose exec -T test drutiny profile:run ci @self "$@"
 
+  debug:
+    usage: Enable debug configuration.
+    cmd: |
+      { ahoy run "php -v|grep -q Xdebug" && echo "Debug is already enabled"; } \
```
Comment (AlexSkrypnyk) [inline .ahoy.yml:134]: despite bing a weird long line, this does multiple things:
- checks that debug is not already enabled and bails early if it is. This is very useful if a developer forgot if xdebug is currently enabled or not.
- enables xdebug and restarts containers (only `php`, `cli` and `test`). Once restarted, check that xdebug actually enabled by examining the output of `php -v`. This is super important as there was an upstream issue where xdebug would be either always enabled or never enabled. This check makes sure that it works as required.

File: docker-compose.yml [hunk 2]
```diff
@@ -23,6 +23,9 @@ x-environment:
     LAGOON_ROUTE: &default-url ${LOCALDEV_URL:-http://govcms8-saas.docker.amazee.io}
     DEV_MODE: ${DEV_MODE:-false}
     X_FRAME_OPTIONS: ${X_FRAME_OPTIONS:-SameOrigin}
+    # Allow to override docker host used from the inside of the containers.
+    DOCKERHOST: ${DOCKERHOST:-}
```
Comment (AlexSkrypnyk) [inline docker-compose.yml:31]: Temporary fix for xdebug.remote_host being incorrectly discovered.
@see https://github.com/amazeeio/lagoon/issues/1088

On some systems, the upstream xdebug remote host discovery mechanism does not have enough time to get the correct host's hostname, leading to incorrect value of ` xdebug.remote_host` to be wrong and IDE not being able to connect to xdebug session. This line adds an ability to provide `DOCKERHOST` explicitly in `.env` file.

**Comment by simesy** [comment]
I would like to test this while working on a project, which will be soon.

**Comment by AlexSkrypnyk** [comment]
@steveworley @srowlands could you please review this one - does not affect production

**Pull request merged**